Index: kcmutils-5.38.0/CMakeLists.txt
===================================================================
--- kcmutils-5.38.0.orig/CMakeLists.txt
+++ kcmutils-5.38.0/CMakeLists.txt
@@ -17,7 +17,7 @@ include(KDEFrameworkCompilerSettings NO_
 include(KDECMakeSettings)
 
 set(REQUIRED_QT_VERSION 5.7.0)
-find_package(Qt5 ${REQUIRED_QT_VERSION} NO_MODULE REQUIRED Widgets DBus Qml Quick QuickWidgets)
+find_package(Qt5 ${REQUIRED_QT_VERSION} NO_MODULE REQUIRED Widgets DBus Qml Quick)
 
 
 include(GenerateExportHeader)
Index: kcmutils-5.38.0/src/CMakeLists.txt
===================================================================
--- kcmutils-5.38.0.orig/src/CMakeLists.txt
+++ kcmutils-5.38.0/src/CMakeLists.txt
@@ -28,7 +28,6 @@ target_link_libraries(KF5KCMUtils
     Qt5::DBus       # dbus usage in kcmoduleproxy.cpp
     Qt5::Qml
     Qt5::Quick
-    Qt5::QuickWidgets
     KF5::CoreAddons
     KF5::I18n
     KF5::IconThemes     # KIconLoader
Index: kcmutils-5.38.0/src/kcmoduleqml.cpp
===================================================================
--- kcmutils-5.38.0.orig/src/kcmoduleqml.cpp
+++ kcmutils-5.38.0/src/kcmoduleqml.cpp
@@ -27,7 +27,6 @@
 #include <QQuickWindow>
 #include <QQuickItem>
 #include <QGuiApplication>
-#include <QQuickWidget>
 
 #include <kdeclarative/kdeclarative.h>
 #include <kquickaddons/configmodule.h>
@@ -107,20 +106,16 @@ void KCModuleQml::showEvent(QShowEvent *
 
     QVBoxLayout* layout = new QVBoxLayout(this);
 
-    QQuickWidget *widget = new QQuickWidget(d->configModule->engine(), this);
-    widget->setResizeMode(QQuickWidget::SizeRootObjectToView);
-    d->quickWindow = widget->quickWindow();
+    d->quickWindow = new QQuickWindow();
     d->quickWindow->setColor(QGuiApplication::palette().window().color());
     connect(qApp, &QGuiApplication::paletteChanged, d->quickWindow, [=]() {
         d->quickWindow->setColor(QGuiApplication::palette().window().color());
     });
 
-    QQmlComponent *component = new QQmlComponent(d->configModule->engine(), this);
-    component->setData(QByteArrayLiteral("import QtQuick 2.3\nItem{}"), QUrl());
-    QObject *root = component->create();
-    widget->setContent(QUrl(), component, root);
-
-    d->configModule->mainUi()->setParentItem(widget->rootObject());
+    //The created widget takes ownership of the QWindow
+    QWidget *widget = QWidget::createWindowContainer(d->quickWindow, this);
+ 
+    d->configModule->mainUi()->setParentItem(d->quickWindow->contentItem());
 
     //set anchors
     QQmlExpression expr(d->configModule->engine()->rootContext(), d->configModule->mainUi(), QStringLiteral("parent"));
